pages/_app.tsx

```diff
@@ -45,7 +45,7 @@ const MainLayout = ({ Component, pageProps }) => {
           site_name: data.title,
           images: [
             {
-              url: 'https://tinacms.org/img/tina-twitter-share.png',
```

Comment:
Is the old image stored on this repo? If so you can remove it :)

Reply:
✅